packages/voila/src/app.ts

```diff
@@ -121,23 +119,23 @@ export class VoilaApp extends JupyterFrontEnd<IShell> {
   /**
    * A promise that resolves when the Voila Widget Manager is created
    */
-  get widgetManagerPromise(): PromiseDelegate<VoilaWidgetManager> {
+  get widgetManagerPromise(): PromiseDelegate<any> {
     return this._widgetManagerPromise;
   }
 
-  set widgetManager(manager: VoilaWidgetManager | null) {
+  set widgetManager(manager: any) {
     this._widgetManager = manager;
     if (this._widgetManager) {
       this._widgetManagerPromise.resolve(this._widgetManager);
     }
   }
 
-  get widgetManager(): VoilaWidgetManager | null {
+  get widgetManager(): any {
```

Comment:
Wondering it this `any` is temporary, or now needed? If so we could likely add a comment to explain the reason?

Reply:
Yes I was about to add a comment about this 👍🏽 

It is needed and I'm not sure we can workaround this for now. The thing is we should really not import anything that imports `@jupyter-widgets/base` to not pollute the shared packages with a wrong ipywidgets version.

Unless there is a way to just import the type for the build to be happy but not the runtime package?

Reply:
`import type` can work?

Reply:
To do this I'd have to make a split of package/extension

Reply:
Actually no